### DISCUSSION TITLE
Self-hostable web dashboard

### DISCUSSION THREAD

**Discussion by ClementCariou** [akiver/cs-demo-manager#711] [Ideas]
Since csdm now runs on PostgreSQL and the UI is coded using Electron, maybe there could be a self hostable web dashboard to view the data from any device (reusing most of the code).

**Comment by RyanBlakeIT** [comment]
Being able to host this would be great.  I have multiple VPSes that I could stand it up on.  Then it could automatically download the demos and, like you said, you could view the information via the web.  The trick would be securing the sensitive information such as the API keys if you wanted to make it accessible from any device (short of using a VPN).

**Comment by rick-gnous** [comment]
3 years later, I agree and reup the post. it can be very interesting to have a service like that with a cron which pull the last games. maybe an account system can be usefull too, to manage the pin players ?
